olive/engine/engine.py

```diff
@@ -910,6 +919,17 @@ def _run_pass(
 
         run_start_time = datetime.now().timestamp()
         try:
+            from olive.passes.onnx.perf_tuning import OrtPerfTuning
+
+            # For perf-tuning, we need the model evaluation happens in target even if it is a pass.
+            # TODO(myguo): consider more better way to handle System doesn't support run pass like DockerSystem
+            if (
+                isinstance(p, OrtPerfTuning)
+                and isinstance(host, LocalSystem)
```

Comment:
Will there be the case: run pass in python evn system and run evaluation in aml system?

Reply:
I am not sure from the time being, I only enable the target from my existing knowledge.